src/types.rs

```diff
@@ -883,3 +883,57 @@ where
     }
     deserializer.deserialize_any(HeightVisitor)
 }
+
+#[derive(Clone, Debug, PartialEq, Serialize, Default)]
+pub struct WalletCreateFundedPsbtOptions {
+    #[serde(rename = "fee_rate", skip_serializing_if = "Option::is_none")]
+    pub fee_rate: Option<f64>,
+    #[serde(rename = "lock_unspent", skip_serializing_if = "Option::is_none")]
+    pub lock_unspent: Option<bool>,
+    #[serde(rename = "conf_target", skip_serializing_if = "Option::is_none")]
+    pub conf_target: Option<u16>,
+    #[serde(rename = "replaceable", skip_serializing_if = "Option::is_none")]
+    pub replaceable: Option<bool>,
+}
+
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct WalletCreateFundedPsbt {
+    pub psbt: String,
+    #[serde(deserialize_with = "deserialize_bitcoin")]
+    pub fee: Amount,
+    #[serde(rename = "changepos")]
+    pub change_pos: i32,
+}
+
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct WalletProcessPsbt {
+    pub psbt: String,
+    pub complete: bool,
+}
+
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct FinalizePsbt {
+    pub psbt: Option<String>,
+    pub hex: Option<String>,
+    pub complete: bool,
+}
```

Comment:
NACK, this should be a `WalletProcessPsbtResult` similar to https://github.com/alpenlabs/bitcoind-async-client/blob/c151abd31f8d47d2ba76282da56217d5fe725827/src/types.rs#L359-L376